src/converters/Mpdf.php

```diff
@@ -43,20 +43,25 @@ protected function convertInternal($html, $outputFileName, $options)
                 'tempDir' => Yii::getAlias(ArrayHelper::remove($options, 'tempDir', '@runtime')),
             ];
 
+            $pdf = new \Mpdf\Mpdf($config);
+
             if (isset($options['fontDir'])) {
-                $config['fontDir'] = Yii::getAlias($options['fontDir']);
+                $pdf->AddFontDirectory(Yii::getAlias($options['fontDir']));
                 unset($options['fontDir']);
             }
 
-            $pdf = new \Mpdf\Mpdf($config);
         } else {
             $pdf = new \mPDF($charset, $pageSize);
         }
 
         foreach ($options as $name => $value) {
             $setter = 'Set' . $name;
             if (method_exists($pdf, $setter)) {
-                $pdf->$setter($value);
+                if (is_array($value)) {
+                    call_user_func_array([$pdf, $setter], $value);
```

Comment:
This is unacceptable: such change may break setup for other setters.

Reply:
are setters that require array as an argument ex:
https://mpdf.github.io/reference/mpdf-functions/setprotection.html